setup.php

```diff
@@ -33,6 +33,8 @@
 
 // Checks if the server's PHP version is compatible with OpenEMR:
 require_once(dirname(__FILE__) . "/common/compatibility/Checker.php");
```

Comment:
TODO for me:
Should be able to remove above 2 lines since the autoload.php below now will load this. This change tested well previously, but will test it again.